### DISCUSSION TITLE
Overwrite fileDialogService.showSaveDialog() programatically?

### DISCUSSION THREAD

**Discussion by MyJumperBroke23** [microsoft/vscode-discussions#1617] [Extension Development QnA] [answered]
Hi, I'm currently trying to build an extension that saves Copilot logs automatically to file. The builtin command to export copilot log has the following
```
const result = await fileDialogService.showSaveDialog({
				defaultUri,
				filters
			});
```
Is there a way to overwrite this programatically, and just directly save a file to a location specified in code?

**Comment by isidorn** [comment, marked as answer]
No, you can not do that from another extension (if I correctly understand what you are doing)
